service/actions/channels.js

```diff
@@ -381,6 +381,11 @@ export function deleteChannel(teamId, channelId) {
             return;
         }
 
```

Comment:
I was under the impression that we want to keep all view related logic out of the service actions.  In the case that  the web-hooks are not working we will need this before delete action to set the current channel.  Open to ideas here if this is not desired.